### PR TITLE
add coverage test for methods.R

### PR DESCRIPTION
**Pull request by MichaelChirico**
Everything uncovered in methods.R is covered by #630

### CODE REVIEW

File: tests/testthat/test-rstudio_markers.R [hunk 1]
```diff
@@ -1,4 +1,3 @@
-context("rstudio_source_markers")
```
Comment (AshesITR) [inline tests/testthat/test-rstudio_markers.R:1]: Why has the test context been removed?

Comment (MichaelChirico) [reply tests/testthat/test-rstudio_markers.R:1]: See https://www.tidyverse.org/blog/2020/10/testthat-3-0-0/ -- it's already kind of vestigial, and will be deprecated from testthat 3.0.0

Comment (MichaelChirico) [reply tests/testthat/test-rstudio_markers.R:1]: If you think it's better to keep the removal of `context()` to a separate PR, lmk. just stripped it while I'm in here.

Comment (AshesITR) [reply tests/testthat/test-rstudio_markers.R:1]: Okay, thanks for the link - missed that.
I think it's okay to remove it gradually.

**Review by AshesITR — APPROVED** [review]

**Review by AshesITR — CHANGES_REQUESTED** [review]
LGTM, please de-lint `test-rstudio_markers.R`.

**Review by AshesITR — APPROVED** [review]

**Pull request merged**